apps/api/v1/services/user.service.ts

```diff
@@ -22,3 +22,11 @@ export async function createUser({ data }: { data: CreateUser }) {
     data,
   })
 }
+
+export async function deleteUser(userId: string) {
+  prisma.user.delete({
```

Comment:
```suggestion
  return prisma.user.delete({
```

Reply:
I updated, just do same with other delete services :D

Reply:
No. Other services always have at least `await` or `return` to ensure they are always returning a promise.